src/libraries/System.Net.Http/src/System/Net/Http/HttpRequestMessage.cs

```diff
@@ -156,14 +156,14 @@ public override string ToString()
                 sb.Append($"{_requestUri}");
             }
 
-            sb.Append("', Version: ");
-            sb.Append(_version);
+            sb.Append($"', Version: {_version}");
 
             sb.Append(", Content: ");
             sb.Append(_content == null ? "<null>" : _content.GetType().ToString());
 
-            sb.AppendLine(", Headers:");
-            HeaderUtilities.DumpHeaders(sb, _headers, _content?.Headers);
+            sb.Append(", Headers:");
+            sb.Append(Environment.NewLine);
```

Comment:
Nit: consider adding an `AppendLine(string)` helper in VSB.

Reply:
My assumption was that the method is missing on purpose to make the type leaner. I would be hesitant to add it without reviewing how applicable it is in other libraries.